source/include/actbl2.h

```diff
@@ -1550,7 +1550,7 @@ enum AcpiMadtLpcPicVersion {
 
 typedef struct acpi_madt_oem_data
 {
-    UINT8                   OemData[];
+    ACPI_FLEX_ARRAY(UINT8, OemData);
```

Comment:
Structs with flexible array members can't be nested inside other structs, only unions, without extensions

Reply:
Yes, this works around a deficiency in the C spec using GNU extensions which have been supported by GCC and Clang for a very long time.

Reply:
I realise that, but you're just swapping one extension for another which doesn't achieve anything as far as I can tell.

If your goal is to avoid extensions then just `typedef UINT8 ACPI_MADT_OEM_DATA[]`, which is an API break but just requires changing `Expr->OemData` to `(*Expr)`.